edib/make-app.mk

```diff
@@ -27,7 +27,7 @@ postinfo:
 release: $(RELEASE_FILE)
 
 $(RELEASE_FILE): app-compile phoenix-assets
-	$(IN_APP_DIR) MIX_ENV=$(MIX_ENV) mix release
+	$(IN_APP_DIR) mix release --env=prod
```

Comment:
I think that's a good idea. It'll prevent breaking in the scenario you described. I'll push the change shortly